boards/arm/beagle_play_cc1352/doc/index.rst

```diff
@@ -0,0 +1,179 @@
+| DIO14 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO15 | CC1352_BOOT  | AM62 GPIO0_13                       |
++-------+--------------+-------------------------------------+
+| DIO16 | CC1352_TDO   | TAG-CONNECT TDO                     |
++-------+--------------+-------------------------------------+
+| DIO17 | CC1352_TDI   | TAG-CONNECT TDI                     |
++-------+--------------+-------------------------------------+
+| DIO18 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO19 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO20 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO21 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO22 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO23 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO24 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO25 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO26 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO27 | LED1         | CC1352_LED1 yellow LED9             |
++-------+--------------+-------------------------------------+
+| DIO28 | LED2         | CC1352_LED2 yellow LED8             |
++-------+--------------+-------------------------------------+
+| DIO29 | RF_PA        | Antenna mux PA enable               |
++-------+--------------+-------------------------------------+
+| DIO30 | RF_SUB1G     | Antenna mux SubG enable             |
++-------+--------------+-------------------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+To flash, disable the existing driver that ties up the serial port and use
+the customized BSL Python script.
+
+1. Ensure the bcfserial driver isn’t blocking the serial port.
+
+   .. code-block:: console
+
+      echo "    fdtoverlays /overlays/k3-am625-beagleplay-bcfserial-no-firmware.dtbo" | sudo tee -a /boot/firmware/extlinux/extlinux.conf
+      sudo shutdown -r now
+
+2. Install CC1352-flasher
+
+   .. code-block:: console
+
+      sudo apt install python3-pip
+      pip3 install cc1352-flasher
+
+3. Flash the CC1352P7
+
+   .. code-block:: console
+
+      west flash
+
+Debugging
+=========
+
+For debugging, you can use the serial port or JTAG. You can use OpenOCD
+over the Tag-Connect header on the board.
```

Comment:
are you able to provide openocd.cfg file like many other boards do?

Reply:
No, I haven't pesonally used tag-connect with this board and couldn't find anyone else on discord who has. Maybe @jadonk can provide some input.

Reply:
Ok, so it does not seem like we can provide `openocd.cfg` yet (since no one seems to have used it for cc1352). I will try to get openocd working when I have time.

Reply:
If it's never been tested, I think you should be dropping the mention to openOCD for now, no?

Reply:
@Ayush1325 , the openocd configs have been in use for a long time and is the default way to load/flash on a CC135X launchpad.
https://sourceforge.net/p/openocd/code/ci/master/tree/tcl/board/ti_cc13x2_launchpad.cfg
https://sourceforge.net/p/openocd/code/ci/master/tree/tcl/target/ti_cc13x2.cfg

Tagconnect is just a connector, you should be able to reuse the same configuration with Tagconnect.

Reply:
@vaishnavachath How does this look? https://github.com/zephyrproject-rtos/zephyr/pull/64718/commits/762f362ffaa57677c76c1d45beccba2d2b91a524
I don't have the jumper cables to test it myself rn.

Reply:
@vaishnavachath does this work?